Boundaries/GCCA/GCCA.json

```diff
@@ -3,7 +3,7 @@
   "properties": {
     "id": "GCCA",
     "prefix": [
-      "GCCA"
+      "GCCA_AAC"
```

Comment:
So the callsign when connected is `GCAA_AAC_APP`?

Reply:
Yes, for this one is GCCA_AAC_APP and it was clarified on its .json as the new position also starts with GCCA (GCCA_INB_APP).